### DISCUSSION TITLE
No Motion events any more since I activated ring subscription?

### DISCUSSION THREAD

**Discussion by Mario084** [tsightler/ring-mqtt#679] [Support Q&A] [answered]
Hello everyone,

the ring-mqtt container was working like a charme combined with my Openhab installation but since I reactivated the ring subscription for video cloud storage I don´t get any motion or ding updates via MQTT.
I´m using Mqtt explorer to check the values but nothing happens here...

I updated my docker container to the latest 5.5 version of ring-mqtt with no success.

Please - any idea how to fix this? Thank you!

**Comment by Mario084** [comment]
I want to add that the Snapshot image is working as before! I get the picture updates in Mqtt...

**Comment by tsightler** [comment, marked as answer]
Did you perform the steps noted in the release notes for v5.3.0 and later to fix notifications when upgrading from previous versions (you didn't say what version you upgraded from or what steps you have taken to correct the issue)?

https://github.com/tsightler/ring-mqtt/blob/main/docs/CHANGELOG.md#v530

**Reply by Mario084** [reply]
Thank you so much for your help! I didn´t realize that there is something to do - I really though that this problem was connected with the ring subscription plan! Now everything is working as before. Thank you
